docs/sources/variables/verify-option-get-from-url.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Verify option get from url"
```

Comment:
```suggestion
title = "Verify option"
```

Reply:
The side menu title should match the h1 title.

Reply:
updated